ui/admin/app/routes/scopes/scope/credential-stores/credential-store.js

```diff
@@ -25,5 +25,10 @@ export default class ScopesScopeCredentialStoresCredentialStoreRoute extends Rou
       into: 'scopes/scope/credential-stores/credential-store',
       outlet: 'navigation',
     });
+
+    this.render('scopes/scope/credential-stores/credential-store/-actions', {
```

Comment:
Comment for this method needs to be updated to handle new actions section. It currently reads as: `Render credential store header and navigation page sections.`.

Reply:
Good catch!!